.gn

```diff
@@ -23,6 +23,10 @@ default_args = {
   use_ozone = false
   use_udev = false
 
+  # TODO(ry) We may want to turn on CFI at some point. Disabling for simplicity
+  # for now. See http://clang.llvm.org/docs/ControlFlowIntegrity.html
```

Comment:
Why is this necessary? "Simplicity" seems like a good argument but I don't see what gets simpler, I see only an added flag that begs for explanation.

Reply:
It fixes the linux release build (which isn't tested in CI) 
https://github.com/denoland/deno/issues/438#issuecomment-410939516